app/auth_routes.js

```diff
@@ -0,0 +1,63 @@
+var express = require('express')
```

Comment:
this is 'boilerplate' but basically where the redirect happens. 
at the moment, this goes to the auth0 login default, and so in effect isn't much different than #2035 

I'd like to play around with this more to see if we can use passport with auth0 but wrapped around our existing login embed form. 

The advantage here might be more when we want to add additional providers like coil, etc. We have a pattern or strategy to follow.